### ISSUE TITLE
Data from cloud not updated

### ISSUE THREAD

**Issue by pbonato** [DurgNomis-drol/mytoyota#369]
With the latest version (and it was the same with the previous version), when I try to make a call to get data from the cloud for my TOYOTA BZ4X, I receive outdated data without any errors.

If I try to open the MyT mobile app (the black one) and refresh the data (I don’t know if you experience the same issue, but it is very slow to respond, and in some cases, the app gets stuck, so I need to force close it), the call with the MyToyota Python module starts working again for a few minutes. However, after that, it goes back to returning old, non-refreshed values.

I’m not sure if this is truly a bug in the module or a problem with the Toyota Cloud. Has anyone else experienced the same issue?

**Comment by thomasattree** [comment]
I've experienced the same behaviour. Smells like an issue with Toyota cloud rather than the integration.

**Comment by OliverAbraham** [comment]
Yeah seems to be a Toyota issue. My Android app data also needs a manual refresh by swiping down.
I also have implemented the protocol in C#, and here's the same. Data is often old.
(https://github.com/OliverAbraham/Abraham.MyToyotaClient)
I seems Toyota API pings my car very rarely.
I haven't found out which parameter or flag is needed to make the API to call my car.

**Comment by CM000n** [comment]
Closed due to discontinuation of the project. See: https://github.com/DurgNomis-drol/mytoyota/issues/404
